dd-sdk-android/src/main/kotlin/com/datadog/android/Datadog.kt

```diff
@@ -76,11 +77,42 @@ object Datadog {
      * @see [DatadogConfig]
      * @throws IllegalArgumentException if the env name is using illegal characters and your
      * application is in debug mode otherwise returns false and stops initializing the SDK
+     * @deprecated Use the [Datadog.initialize] instead which requires a privacy [Consent] parameter.
+     */
+    @Deprecated(
+        "This method is deprecated and uses the [Consent.GRANTED] " +
+            "flag as a default privacy consent.This means that the SDK will start recording " +
+            "and sending data immediately after initialisation without waiting " +
+            "for the user's privacy consent.",
+        ReplaceWith(
+            expression = "Datadog.initialize(context, Consent.PENDING, config)",
+            imports = ["com.datadog.android.privacy.Consent"]
+        )
+    )
+    @Suppress("LongMethod")
+    @JvmStatic
+    fun initialize(
+        context: Context,
+        config: DatadogConfig
+    ) {
+        initialize(context, Consent.GRANTED, config)
+    }
+
+    /**
+     * Initializes the Datadog SDK.
+     * @param context your application context
+     * @param config the configuration for the SDK library
+     * @param privacyConsent as the initial state of the Privacy consent flag.
```

Comment:
Dunno if it's important, but those params are listed in different order in the actual signature.